packages/core/package.json

```diff
@@ -25,15 +25,17 @@
     "prebuild": "npm run clean",
     "postbuild": "node scripts/copyFiles.js",
     "clean": "rimraf build",
-    "publish": "npm publish build"
+    "publish": "npm publish build",
+    "test": "jest"
   },
   "dependencies": {
     "@material-ui/core": "^3.0.0",
     "@material-ui/icons": "^3.0.0",
     "classnames": "^2.2.5",
-    "lodash": "^4.17.4",
+    "lodash": ">=4.17.12",
     "prop-types": "^15.5.7",
     "react-aria-menubutton": "^4.3.1",
+    "react-test-renderer": "^16.12.0",
```

Comment:
Do we want to yoink this guy, since we aren't using it now?

Reply:
Sounds good to me, we will probably want to go with a different testing framework when we get to that point anyway